packages/studio-web/src/app/app.component.html

```diff
@@ -1 +1,2 @@
 <router-outlet></router-outlet>
+<div class="version">&#64;readalongs/studio-web version:{{ version }}</div>
```

Comment:
Trivial suggestion...
```suggestion
<div class="version">&#64;readalongs/studio-web version: {{ version }}</div>
```
maybe?